doc/api_assets/script.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const eqIsh = (a, b, fuzz = 2) => {
```

Comment:
We need to scope this to avoid leeaking to the global scope.
```suggestion
{
  const eqIsh = (a, b, fuzz = 2) => {
```